test/unit/rules/no-down-event-binding-test.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const { ERROR_MESSAGE } = require('../../../lib/rules/no-down-event-binding');
+const generateRuleTests = require('../../helpers/rule-test-harness');
+
+generateRuleTests({
+  name: 'no-down-event-binding',
+
+  config: true,
+
+  good: [
+    "<div {{on 'keyup' this.doSomething}}></div>",
+    "<div {{action this.doSomething on='keyup'}}></div>",
+  ],
+
+  bad: [
```

Comment:
What about components?

```hbs
<MyComponent @keyDown={{@close}}>
```

```hbs
{{my-component keyDown=foo}}
```

Reply:
Hmm, that's an interesting idea. That could handle "classic component" arguments that automatically become event listeners, but that's increasingly irrelevant for Glimmer components that don't use that behavior. 

If we think about the mental model going forward and assume these arguments do not automatically become event handlers, would we want to lint random component arguments? It would probably be safe to assume that an argument called `keyDown` maps to such an event listener, but would that be assuming too much from the linter's perspective?

Reply:
I'd be fine excluding components as the default behavior. If someone wanted to follow-up later, they could add a rule option `catchComponents` (default false) to be more aggressive. 

For now, could you add these as valid test cases then?

Reply:
Sure, I'll add those tests, too!